Source/JavaScriptCore/heap/MarkedBlockInlines.h

```diff
@@ -309,7 +309,9 @@ void MarkedBlock::Handle::specializedSweep(FreeList* freeList, MarkedBlock::Hand
     size_t count = 0;
     uintptr_t secret = static_cast<uintptr_t>(vm.heapRandom().getUint64());
     bool isEmpty = true;
-    Vector<size_t> deadCells;
+    static_assert(UINT16_MAX <= atomsPerBlock);
+    static_assert(atomsPerBlock <= 2048); // Arbitrary size limit so the vector doesn't get too big.
```

Comment:
These two things can't be true at the same time, right?

Reply:
Oh whoops! That first one is reversed, good catch. I need to adjust the other assertion too to support platforms with 64k pages, so I'll push an amendment shortly.